Mathlib/Data/PNat/Basic.lean

```diff
@@ -278,6 +278,21 @@ theorem pow_coe (m : ℕ+) (n : ℕ) : ↑(m ^ n) = (m : ℕ) ^ n :=
   rfl
 #align pnat.pow_coe PNat.pow_coe
 
+/-- b is greater one if any a is less than b -/
+theorem one_lt_of_lt {a b: ℕ+} (hab: a < b): 1 < b := by
```

Comment:
Please put spaces around the colon. (Did you see the [style guide](https://leanprover-community.github.io/contribute/style.html) already? It explains this and many other things - by reading and following it, you can make your code get accepted faster - as reviewers don't have to point this out :-))
```suggestion
theorem one_lt_of_lt {a b : ℕ+} (hab : a < b) : 1 < b := by